docs/examples/substrate/call_flags.sol

```diff
@@ -0,0 +1,23 @@
+contract CallFlags {
+
+    enum CallFlag { FORWARD_INPUT, CLONE_INPUT, TAIL_CALL, ALLOW_REENTRY }
+
+    // Set all flags found in _flags to true.
+    function bitflags(CallFlag[] _flags) internal pure returns (uint32 flags) {
+        for (uint n = 0; n < _flags.length; n++) {
+            flags |= (2 ** uint32(_flags[n]));
+        }
+    }
+
+    // Call the contract at _address with the given _selector.
+    // Specify any flag used for the contract call in _flags.
+    function call_with_flags(
+        address _address,
```

Comment:
Why do you prefer arguments that start with an underscore?
(I'm not asking you to change. It is just my curiosity.)

Reply:
I thought it was in the Solidity style guide because I saw that a lot. But I checked, it's not. It seems to be something many Solidity devs just do.

However in this case for `_address` I left it in place because `address`  is already the name of the type...